public/pages/utils/anomalyResultUtils.ts

```diff
@@ -575,6 +578,23 @@ export const parseBucketizedAnomalyResults = (result: any): Anomalies => {
         get(rawAnomaly, 'anomaly_grade') !== undefined &&
         get(rawAnomaly, 'feature_data', []).length > 0
       ) {
+        const featureDataMap: {[key: string]: string} = {};
+        get(rawAnomaly, 'feature_data', []).filter(element => get(element, 'feature_id') && get(element, 'feature_name')).
+        forEach((feature: any) => {
+          featureDataMap[get(feature, 'feature_id')] = get(feature, 'feature_name');
+        });
+        const relevantAttributionMap: {[key: string]: number} = {};
+        get(rawAnomaly, 'relevant_attribution', []).filter(element => get(element, 'feature_id') && get(element, 'data')).
+        forEach((attribution: any) => {
+          relevantAttributionMap[get(attribution, 'feature_id')] = get(attribution, 'data');
+        });
+        const contributions: { [key: string]: FeatureContributionData } = {}
+        for (const [key] of Object.entries(featureDataMap)) {
+          contributions[key] = {
+            name: featureDataMap[key],
+            attribution: relevantAttributionMap[key]
+          }
+        }
```

Comment:
nit: you can make this type-safe by adding `as FeatureContributionData` at the end.